com.unity.multiplayer.mlapi/Tests/Runtime/NetworkPrefabHandlerTests.cs

```diff
@@ -0,0 +1,166 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using MLAPI.Spawning;
+using NUnit.Framework;
+
+
+
+namespace MLAPI.RuntimeTests
+{
+    /// <summary>
+    /// The NetworkPrefabHandler unit tests validates:
+    /// Registering with GameObject, NetworkObject, or GlobalObjectIdHash
+    /// Newly assigned rotation or position values for newly spawned NetworkObject instances are valid
+    /// Destroying a newly spawned NetworkObject instance works
+    /// Removing a INetworkPrefabInstanceHandler is removed and can be verified (very last check)
+    /// </summary>
+    public class NetworkPrefabHandlerTests
+    {
+        [Test]
+        public void NetworkPrefabHandlerClass()
+        {
+            //Only used to create a network object based game asset
+            Assert.IsTrue(NetworkManagerHelper.StartNetworkManager());
+
+            Guid baseObjectID = NetworkManagerHelper.AddGameNetworkObject("NetworkPrefabHandlerTestObject");
```

Comment:
maybe extract this string to a local variable instead of hardcoding it? (we're using it a few times below)

```suggestion
            var testObjectName = "NetworkPrefabHandlerTestObject";
            Guid baseObjectID = NetworkManagerHelper.AddGameNetworkObject("NetworkPrefabHandlerTestObject");
```